src/primitives/transaction.h

```diff
@@ -375,6 +383,8 @@ struct CMutableTransaction
     std::vector<CTxOut> vout;
     uint32_t nLockTime;
     std::vector<CPourTx> vpour;
+    uint256 joinSplitPubKey;
+    boost::array<unsigned char, 64> joinSplitSig;
```

Comment:
Make this ``joinsplit_sig_t`` as in ``CTransaction``. (It may be useful to move the definition of ``joinsplit_sig_t`` so that it can be referred to by both classes.)